config_fr.xml

```diff
@@ -2,7 +2,7 @@
 <module>
```

Comment:
Could you also remove this file?
Only config.xml is required, others are not.

Thank you,
Kind regards.